tests/celery/test_tasks.py

```diff
@@ -33,11 +33,21 @@ def test_sanitise_and_upload_valid_letter(mocker, client):
 
     sanitise_and_upload_letter('abc-123', 'filename.pdf')
 
-    mock_upload.assert_called_once_with(
-        filedata=mocker.ANY,
-        region=current_app.config['AWS_REGION'],
-        bucket_name=current_app.config['SANITISED_LETTER_BUCKET_NAME'],
-        file_location='filename.pdf',
+    mock_upload.assert_has_calls(
+        [
+            call(
+                filedata=mocker.ANY,
+                region=current_app.config['AWS_REGION'],
+                bucket_name=current_app.config['SANITISED_LETTER_BUCKET_NAME'],
+                file_location='filename.pdf',
+            ),
+            call(
+                filedata=mocker.ANY,
```

Comment:
are we able to replace this mocker.ANY call with `valid_file`?

Reply:
lets see :3

Reply:
done ☺️